src/chatshare.py

```diff
@@ -4,8 +4,23 @@
 Chatshare - A chat sharing application.
 """
 
+import os
+
 from src.docker_manager import get_containers
 
+REQUIRED_ENV_VARS = [
+    'PANEL_API_URL',
+    'PANEL_APPLICATION_KEY',
+    'PANEL_CLIENT_KEY',
+    'DISCORD_TOKEN',
+    'DISCORD_CHANNEL',
+]
+
+for var in REQUIRED_ENV_VARS:
+    value = os.environ.get(var)
+    if not value:  # catches None and empty string
```

Comment:
would this also catch 0? not that it's an issue at the moment, just a consideration.